src/lib.rs

```diff
@@ -97,6 +98,18 @@ pub trait Platform: Sized {
         }
     }
     #[inline(always)]
+    fn wasm32(self, _input: Self) -> Self {
+        #[cfg(target_arch="wasm32")]
+            {
```

Comment:
Can you fix the whitespace?  Looks like you used tabs

Reply:
They are showing up as spaces... I'm confused.